Shared/Samples/Show grid/ShowGridView.swift

```diff
@@ -251,15 +251,17 @@ private extension ShowGridView {
 }
 
 private extension ArcGIS.Grid.LabelPosition {
-    static var allCases: [Self] = [
-        .allSides,
-        .center,
-        .topLeft,
-        .topRight,
-        .bottomLeft,
-        .bottomRight,
-        .geographic
-    ]
+    static var allCases: [Self] {
+        return [
+            .allSides,
+            .center,
+            .topLeft,
+            .topRight,
+            .bottomLeft,
+            .bottomRight,
+            .geographic
+        ]
```

Comment:
```suggestion
        [
            .allSides,
            .center,
            .topLeft,
            .topRight,
            .bottomLeft,
            .bottomRight,
            .geographic
        ]
```

Reply:
<img width="564" alt="image" src="https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/9660181/ea07b9d2-ee39-46a6-b2a6-930467ea29fb">

Without return it would trigger a linter warning. Similar to [this](https://github.com/Esri/arcgis-maps-sdk-swift-samples/pull/434#discussion_r1664910339)